ext/fluxnet_simulations/forcing.jl

```diff
@@ -436,8 +443,9 @@ function FluxnetSimulations.get_comparison_data(
     val = -9999,
     timestamp_end = true,
 )
-    fluxnet_csv_path = ClimaLand.Artifacts.experiment_fluxnet_data_path(site_ID)
-    (data, columns) = readdlm(fluxnet_csv_path, ','; header = true)
+    # Read data and process timestamps
+    (data, columns, local_datetime, UTC_datetime) =
+        read_fluxnet_data(site_ID; hour_offset_from_UTC)
```

Comment:
does the same thing happen here? we overwrite local and UTC datetime later on (to use the TIMESTAMP_END)